<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgDemo71</class>
 <widget class="QDialog" name="dlgDemo71">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Consulta de Gran Cantidad de Registros</string>
  </property>
  <widget class="QPushButton" name="btnConsultarCubso">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Consultar Cubso</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="pbrConsulta">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>951</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QTableWidget" name="tblCubso">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>961</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPrimero">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>550</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExportarPDF">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Exportar PDF</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAnterior">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>550</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSiguiente">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>550</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUltimo">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>550</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtTotalPaginas">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>550</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="cboPaginaActual">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>550</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblMensaje">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>20</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
